trade-finance-manager-api/src/v1/controllers/deal.tasks.api-test.js

```diff
@@ -41,16 +41,45 @@ describe('createDealTasks', () => {
   });
 
   describe('shouldCreatePartiesTask', () => {
-    describe('when a deal has tfm.exporter.partyUrn', () => {
+    describe('when a deal has tfm.exporter.partyUrn but NO tfm.buyer.partyUrn and deal is BSS', () => {
+      it('should return true', () => {
+        const result = shouldCreatePartiesTask(mockDealWithPartyUrn);
+        expect(result).toEqual(true);
+      });
+    });
+
+    describe('when a deal does NOT have tfm.exporter.partyUrn but has tfm.buyer.partyUrn and deal is BSS', () => {
+      it('should return true', () => {
+        mockDealWithPartyUrn.tfm.parties.buyer = { partyUrn: 'Test' };
+        mockDealWithPartyUrn.tfm.parties.exporter = null;
```

Comment:
Instead of null can it be just empty as that will be default.

Reply:
done